src/modules/home/components/single-property/single-property-review.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Rating from '@mui/material/Rating';
+import StarIcon from '@mui/icons-material/Star';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import { SinglePropertyReviewStyles } from './single-property-review.styles';
+
+interface Review {
+  id: string;
+  name: string;
+  comment: string;
+  rating: number;
+}
+
+interface SinglePropertyReviewProps {
+  reviews: Review[];
+}
+
+function SinglePropertyReview({ reviews }: SinglePropertyReviewProps) {
+  const [numReviews, setNumReviews] = useState(12);
+
+  const handleShowMore = () => {
+    setNumReviews((prev) => prev + 12);
+  };
+
+  const overallRating = reviews.reduce((acc, review) => acc + review.rating, 0) / reviews.length;
```

Comment:
This must be calculated and sent from the backend. Because not all reviews are sent at once, there will be a limit to sending it in chunks. Imagine there are 1000+ reviews....